internal/provider/resource_streaming_tenant.go

```diff
@@ -191,6 +191,11 @@ func resourceStreamingTenantRead(ctx context.Context, resourceData *schema.Resou
 	if err != nil {
 		return diag.Errorf("failed to get streaming tenant: %v", err)
 	}
+	if getTenantResponse.HTTPResponse.StatusCode == http.StatusNotFound {
```

Comment:
Nit: should `http.StatusNotFound` be changed to `404` to be consistent with other checks?

Reply:
I would actually prefer `StatusNotFound` as opposed to using "magic numbers", but I'll make it all consistent.